bin/run-tests.sh

```diff
@@ -24,7 +25,7 @@ for test_dir in tests/*; do
     rm -rf "${test_dir_path}/target"
     rm -f "${test_dir_path}/Cargo.lock"
 
-    bin/run.sh "${test_dir_name}" "${test_dir_path}" "${test_dir_path}" > /dev/null 2>&1
```

Comment:
I removed this initially for debugging purposes, but I didn't put it back because the run script isn't supposed to write to stdout / stderr anyway. Suppressing it only leads to more confusion when something does go wrong.